lib/child_process.js

```diff
@@ -399,7 +400,7 @@ function normalizeSpawnArguments(file, args, options) {
   if (Array.isArray(args)) {
     args = args.slice(0);
   } else if (args !== undefined &&
-             (args === null || typeof args !== 'object')) {
+    (args === null || typeof args !== 'object')) {
```

Comment:
Can you undo unrelated style changes? They make review harder, break `git blame`, and tend to induce merge conflicts.